### PR TITLE
Update rubygem-foreman_openscap to 5.2.1

### PR DESCRIPTION
**Pull request by lzap**
For plugin updates, please indicate which repos this should be built into:

* Nightly
* 3.1

There is no debian package I assume.

DO NOT MERGE

@ron-lavi can you review the JS stack changes? Looks weird to me why the packaging script would want to remove graphql, maybe it is missing from the `package.json`? Should we add them?

### CODE REVIEW

File: packages/plugins/rubygem-foreman_openscap/rubygem-foreman_openscap.spec [hunk 1]
```diff
@@ -36,10 +36,8 @@ BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
 BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 8.4.1
 BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 9.0.0
-BuildRequires: %{?scl_prefix}npm(graphql-tag) >= 2.11.0
-BuildRequires: %{?scl_prefix}npm(graphql-tag) < 3.0.0
-BuildRequires: %{?scl_prefix}npm(graphql) >= 15.5.0
-BuildRequires: %{?scl_prefix}npm(graphql) < 16.0.0
```
Comment (Ron-Lavi) [inline packages/plugins/rubygem-foreman_openscap/rubygem-foreman_openscap.spec:42]: For now let's keep `graphql-tag` requirement here as it's not in SCAP package.json or foreman-js

`graphql` is already part of Foreman-js: https://github.com/theforeman/foreman-js/blob/master/packages/vendor-core/package.json#L51 so it should work without it 🤞🏼

**Review by Ron-Lavi — COMMENTED** [review]
Thanks for the PR @lzap !

File: packages/plugins/rubygem-foreman_openscap/rubygem-foreman_openscap.spec (hunk 1 above)
Comment (lzap) [reply packages/plugins/rubygem-foreman_openscap/rubygem-foreman_openscap.spec]: Did that

**Comment by lzap** [comment]
However it does not build:

    DEBUG util.py:600:  No matching package to install: 'npm(@theforeman/builder) < 9.0.0'

**Comment by Ron-Lavi** [comment]
@evgeni can you take a look? 🙏🏼

**Comment by evgeni** [comment]
No, but you could ask @theforeman/packaging, which will tell you that `{builder/vendor}` was upgraded to 10 in #7487 and this plugin is still limiting the upper bound of the dependency. :woman_shrugging:

**Comment by evgeni** [comment]
And if you mean the last repoclosure failures, I don't know, ask @theforeman/packaging

**Comment by lzap** [comment]
@Ron-Lavi I am afraid I am lost in this, nothing works, discovery package.json is broken, bootdisk the same, scap the same. We need to rethink this. https://community.theforeman.org/t/whats-up-with-package-json-in-plugins/27145

If you let me exactly what to update here, I can update the PR tomorrow, but then I am off.

**Comment by Ron-Lavi** [comment]
@lzap you can copy the small change I made in #7543, the tests there are green 💚

**Comment by evgeni** [comment]
[test rpm]

The failure is unrelated to the changes

**Comment by Ron-Lavi** [comment]
Thanks @evgeni , closing mine then 🤣

**Comment by lzap** [comment]
I did nothing and it's green now. God help us!

**Comment by ezr-ondrej** [comment]
> I did nothing and it's green now. God help us!

Did you mean @evgeni helped us? :)

File: packages/plugins/rubygem-foreman_openscap/rubygem-foreman_openscap.spec [hunk 2]
```diff
@@ -34,12 +34,12 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
...
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 8.4.1
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 9.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 10.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 11.0.0
 BuildRequires: %{?scl_prefix}npm(graphql-tag) >= 2.11.0
 BuildRequires: %{?scl_prefix}npm(graphql-tag) < 3.0.0
-BuildRequires: %{?scl_prefix}npm(graphql) >= 15.5.0
-BuildRequires: %{?scl_prefix}npm(graphql) < 16.0.0
```
[the remark this answers is not in the document]
Comment (evgeni) [reply packages/plugins/rubygem-foreman_openscap/rubygem-foreman_openscap.spec:42]: Nah, we added that dep to core, so it's not needed here anymore.

**Review by evgeni — APPROVED** [review]

**Comment by evgeni** [comment]
I didn't, Odilon fixed it, I just rekicked the CI.

**Pull request merged**